requirements/constraints.txt

```diff
@@ -19,5 +19,5 @@ pylint==1.9.3
 pycodestyle==2.5.0
 
 #supported in django 1.11
-jsonfield==2.0.2
+jsonfield2==4.0.0.post0
```

Comment:
what is post0?

Reply:
@andrey-canon as @mariajgrimaldi mentioned, jsonfield2 3.0.3 does not support python2.7 
Should we have requirements for python2.7 and requirements for python3.5?